frontend/public/components/app.jsx

```diff
@@ -213,11 +213,11 @@ setInterval(() => store.dispatch(UIActions.updateTimestamps(Date.now())), 10000)
 fetchSwagger();
 
 // Used by GUI tests to check for unhandled exceptions
-window.windowError = false;
+window.windowError = null;
 window.onerror = window.onunhandledrejection = (e) => {
   // eslint-disable-next-line no-console
   console.error('Uncaught error', e);
-  window.windowError = e || true;
+  window.windowError = e || null;
```

Comment:
No reason to use `|| null`.

```suggestion
  window.windowError = e;
```

Reply:
ok, fixed